exporter/sentryexporter/interfaces.go

```diff
@@ -40,3 +42,20 @@ type SentrySpan struct {
 func (s SentrySpan) IsRootSpan() bool {
 	return s.ParentSpanID == ""
 }
+
+// TraceContext describes the context of the trace.
+type TraceContext struct {
+	TraceID     string `json:"trace_id"`
+	SpanID      string `json:"span_id"`
+	Op          string `json:"op,omitempty"`
+	Description string `json:"description,omitempty"`
+}
+
+// SentryTransaction describes a Sentry Transaction.
+// TODO: generate extra fields when creating envelope EventID, Type, User, Platform, SDK
+type SentryTransaction struct {
+	*sentry.Event
+	StartTimestamp time.Time     `json:"start_timestamp,omitempty"`
+	TraceContext   TraceContext  `json:"contexts,omitempty"`
```

Comment:
`omitempty` doesn't apply to structs.

```suggestion
	StartTimestamp time.Time     `json:"start_timestamp"`
	TraceContext   TraceContext  `json:"contexts"`
```

TraceContext should not replace other contexts, let's tackle this over a call?

Reply:
Note: I have changed this functionality in #5